tools/IoTEdgeDevOps/DevOpsLib/BugManagement.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DevOpsLib
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Flurl;
+    using Flurl.Http;
+    using Newtonsoft.Json.Linq;
+
+    public class BugManagement
+    {
+        const string WorkItemPathSegmentFormat = "{0}/{1}/{2}/_apis/wit/wiql/{3}";
+
+        readonly DevOpsAccessSetting accessSetting;
+
+        public BugManagement(DevOpsAccessSetting accessSetting)
+        {
+            this.accessSetting = accessSetting;
+        }
+
+        /// <summary>
+        /// This method is used to execute a Dev Ops work item query and get a list of bugs. 
+        /// If result is not found for a query Id, it will return an entity with no result.
+        /// Note: there is no validation of work item query ids.
+        /// Reference: https://docs.microsoft.com/en-us/rest/api/azure/devops/wit/wiql/get?view=azure-devops-rest-5.1
+        /// </summary>
+        /// <param name="bugQueryId">bug query id from azure dev ops shared queries</param>
+        /// <returns>Number of bugs output by query</returns>
+        public async Task<int> GetBugsQuery(string bugQueryId)
```

Comment:
should it be Task<uint> or Task<ushort>?  if you are going to use unsigned, please make sure to use the right type in sql table, https://social.msdn.microsoft.com/Forums/en-US/ff08c190-a981-4896-9542-3f64b95a84a2/sql-server-data-type-for-signedunsigned-integral-c-types?forum=adodotnetdataproviders.

Reply:
Since SQL server doesn't support uint I feel like it is cleanest to use the same data type everywhere. If you disagree I can change.

Reply:
Not a big concern to me since this is a internal tool.